### PR TITLE
Minor Quick Start update

### PR DESCRIPTION
**Pull request by janainawittner**
I made some stylistic changes but some content changes too.

### CODE REVIEW

**Comment by janainawittner** [comment]
@antmonnier, Please have a look before you publish as this is a process I don't know much about. I changed the content when I thought it was necessary to improve understanding. 
Some examples: 
- "uniquely" is not widely used in EN. Also, it did not fit in context. 
- (Optional), I think is best at the end of the title rather than the start....
- etc.

File: content/developers/en/quick-start.md [hunk 1]
```diff
@@ -43,15 +43,15 @@ Download the client secret JSON, which should look like:
 }
 ```
 
-The client's **id** and **secret** uniquely identify your application. You will generally need to create a single client for your application: The same client can be used by several HubRise accounts.
+The client's **id** and **secret** is used to identify your application. You will generally need to create a single client for your application. The same client can be used by several HubRise accounts.
```
Comment (atorin) [inline content/developers/en/quick-start.md:46]: I think it's better to say "are used to identify".

File: content/developers/en/quick-start.md [hunk 2]
```diff
@@ -151,20 +151,20 @@ Now you can try to place an order with your first application. You can use the s
...
 In some circumstances, a webhook cannot be set up because a callback URL cannot be exposed. This is typical for applications that users download and run from their computer.
 
-In this case, you can set up a _passive callback_: your application can check for new events at regular intervals of time and perform the necessary operations when a new event is found.
+In this case, you can set up a passive callback. Your application will then check for new events at regular intervals of time and perform the necessary operations when a new event is found.
 
 The `Callback: Get Events` request in Postman is an example of how you can do that. Simply create a new order, as usual, and play this request right after to obtain the order's details.
 
 After you process the event, you should delete it so that it will not appear in the next `GET /events` request.
-Unlike active callbacks, in fact, events need to be manually deleted after being processed with a passive callback.
+Unlike active callbacks events need to be manually deleted after being processed with a passive callback.
```
Comment (atorin) [inline content/developers/en/quick-start.md:159]: I think you still need a comma:
"Unlike active callbacks, events need to be manually deleted after being processed with a passive callback."

File: content/developers/en/quick-start.md (hunk 1 above)
Comment (antmonnier) [reply content/developers/en/quick-start.md]: I was reviewing at the exact same time as you!

What about simply "`client_id` identifies your application." ?

File: content/developers/en/quick-start.md (hunk 2 above)
Comment (antmonnier) [reply content/developers/en/quick-start.md:159]: Will change now

File: content/developers/en/quick-start.md (hunk 1 above)
Comment (antmonnier) [reply content/developers/en/quick-start.md]: Never mind :) Will use your version

**Pull request merged**